Makefile

```diff
@@ -28,6 +28,8 @@ SMTP_PASS ?= ''
 SMTP_PORT ?= ''
 SMTP_FROM ?= ''
 ROLE_ARN ?= "arn:aws:iam::485026278258:role/12345"
+TMP_S3_ACCESS_KEY_ID ?= "123"
```

Comment:
Could remove the use of `TMP_` from the env vars and yaml keys.

Reply:
Done. Thank you